content/_releases/22/36.md

```diff
@@ -0,0 +1,136 @@
+you navigate to Integrate > Credentials page you will see a similar view:
+
+{% include img.html max-width="80%" url="/assets/img/RN/2236/RN2236-credentials-page.png" title="Credentials Page" %}
+
+When you select a component your screen would look like the one below for Salesforce:
+
+{% include img.html max-width="80%" url="/assets/img/RN/2236/RN2236-credential.png" title="Component Credential" %}
+
+You can select 3 dots to open the menu to **Verify**, **Edit** or **Delete** the credentials
+or use **+** (plus) in the right above to create a new one.
+
+{% include img.html max-width="80%" url="/assets/img/RN/2236/RN2236-credential-mod.png" title="Credential Modification" %}
+
+Here is how each credential page looks when you Edit it:
+
+{% include img.html max-width="80%" url="/assets/img/RN/2236/EN2236-credential-edit.png" title="Edit the Credential" %}
+
+### Retrieve new sample button
+
+While designing the flow using the flow designer often we need to retrieve new
+sample for the step after making modifications. To improve the user experience
+we added a new button **Retrieve new sample from component**.
+
+{% include img.html max-width="80%" url="/assets/img/RN/2236/RN2236-retrieve-new-sample-button.png" title="Retrieve new sample button" %}
+
+You can now click to retrieve new sample instead of navigating **Back** to do this.
+
+### Long lines in Code component
+
+We improved the user interface of the Node.js Code component configuration field.
+Now you can use long lines of code witout spaces and the interface will wrapp it for you.
+
+{% include img.html max-width="80%" url="/assets/img/RN/2236/RN2236-code-wrap.png" title="Code wrapping." %}
+
+
+{% if site.data.tenant.name == "elastic.io" %}
+
+## HELM3
+
+We constantly improve different aspects of HELM3 deployments in collaboration
+with our partners. This section lists updates and improvements included in this release.
+
+
+
+## OEM Related
+
+Information in this section is intended for our customers who use the
+[OEM version of the elastic.io platform](https://www.elastic.io/saas-embedded-integration/).
+
+### Registration page improvement
+
+If your tenant is using the registration page and the `email_verification` parameter
+is enabled (set to `true`) then only the email address field will show at first.
+
+{% include img.html max-width="80%" url="/assets/img/RN/2236/RN2236-registration-step1.png" title="Registration Page: Step 1 - Enter your email address." %}
+
+When you enter your email address and press **Register** then an email is sent to
+your address with a special link to follow to verify your email address. When you
+follow the link you are presented with the second step to enter a password:
+
+{% include img.html max-width="80%" url="/assets/img/RN/2236/RN2236-registration-step2.png" title="Registration Page: Step 2 - Enter the password." %}
+
+{% endif %}
+
+## Fixed bugs
+
+* Now Gendry creates missing RabbitMQ org users on each run if data lost.
```

Comment:
Errr - this is impossible for me to understand. Suggest we slack call for 2 min, then I can write an explanation